index.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="stylesheet" href="main.css" />
+    <title>Document</title>
+  </head>
+  <body>
+    <header>
+      <nav class="navigator">
+        <h2>Welcome</h2>
+        <img src="images/more.png" alt="more" />
+      </nav>
+    </header>
+    <main>
+      <div class="main-title">
+        <div>
+          <h2 class="main-header">
```

Comment:
- [ ] I kindly suggest that you change this `h2` tag to an `h1` tag because it is your main title and it makes it more [semantic](https://developer.mozilla.org/en-US/docs/Glossary/Semantics)